src/api/Alert/types.ts

```diff
@@ -302,3 +303,8 @@ export interface ICasePreview extends Omit<IFullCase, 'statusChangedOn' | 'statu
 export interface IGetAllAlertUsersParams extends IDataSourceParams {
   mutualAlertIds?: number[];
 }
+
+export interface IBulkAssignCasesParams extends IDataSourceParams {
```

Comment:
Consider moving that type next to `IAssignCasesParams` if you move the bulkAssignCase function, so that the orders of the declarations in index.ts and type.ts are consistent.